public/dist/css/parent.css

```diff
@@ -0,0 +1,28 @@
+.Inner_Data_Container {
```

Comment:
I understand the intended format of the page and what the Inner and Outer containers indicate; but I do not know what information is held in outer versus inner containers. I think the way you have this formatted is correct if where the container is located is more important than for information is contained. But if what information is in Inner/Outer would help readability maybe look into different class names for inner/outer.